run-command.c

```diff
@@ -1407,6 +1407,17 @@ int run_hook_argv(const char *const *env, const char *name,
 	const char *p;
 
 	p = find_hook(name);
+	/*
```

Comment:
Hopefully this is in a separate commit as well.

Reply:
yeah, this wasn't here when i posed the PR.  it looks like the merge target branch moved (force updated).  I'll need to rebase to keep this from appearing in my PR.